packages/patternlab/styleguide/source/_patterns/02-molecules/card.twig

```diff
@@ -5,6 +5,7 @@
 {% set imgClass = card.usage ? card.usage : card.id %}
 {% set detailsClass = card.usage ? " ma__card__details--" ~ card.usage : "" %}
 
+<a class="ma__card__wrapper" href="{{ card.link }}">
```

Comment:
I think moving the wrapper out broke the primary card, can you please fix it? The image should be side-by-side with the next not stacked https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-38060_add_image_option_card_group/?p=organisms-campaign-feature